driver-service/pom.xml

```
@@ -101,6 +106,18 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.liquibase</groupId>
+                <artifactId>liquibase-maven-plugin</artifactId>
+                <version>${liquibase-maven-plugin}</version>
+                <configuration>
+                    <changeLogFile>src/main/resources/db/changelog/db.changelog-master.xml</changeLogFile>
```

Должно быть достаточно:
```suggestion
                    <changeLogFile>db/changelog/db.changelog-master.xml</changeLogFile>
```